weblate/vendasta/access.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from weblate.auth.models import Group
+
+
+def set_permissions(strategy, backend, user, details, **kwargs):
+    """
+    Add all users to Viewers, add developers to 'Vendasta Internal', and add namespaced users to namespace group.
+    """
+    groups_to_add = [Group.objects.get('Viewers')]
+
+    roles = details.get('roles', [])
+    if 'developer' in roles:
+        groups_to_add.append(Group.objects.get(name='Vendasta Internal'))
+    if 'partner' in roles:
+        groups_to_add.append(Group.objects.get(name='Partner Users'))
```

Comment:
Are developers supposed to be in both the 'Vendasta Internal' and 'Partner Users' roles? (I'm assuming the roles match the personals listed at https://iam-prod.apigateway.co/personas)

Reply:
The keys for personas are listed here https://github.com/vendasta/IAM/blob/master/pkg/subject/persona/persona_types.go

I think it's acceptable for developers to be in both, but it probably doesn't make sense to overlap, given that Vendasta Internal grants access to partner components. I'll add an elif.